status-im/status-react
# Install all dependencies on Linux

Fixes #2548 

### Summary:

This merges in Linux-specific setup to the `scripts/setup` script, and renames it from `setup-osx` to `setup`.

### Review notes:

* This only is tested in Ubuntu 16.04 LTS Desktop
* I haven't implemented installing the Android SDK, as this is hairier than anticipated. I think that might be better done in a separate PR. That install in particular seems like the worst one to deal with, and I think iterating on this to get a solid Android setup would be good.

### Steps to test:

1. Clone the repo on ubuntu.
2. Run `scripts/setup`
3. It should work!

status: ready
